components/gfx/font_cache_task.rs

```diff
@@ -198,15 +218,15 @@ impl FontCache {
         for_each_available_family(|family_name| {
             let family_name = LowercaseString::new(&family_name);
             if !self.local_families.contains_key(&family_name) {
```

Comment:
It seems inefficient to create a new LowercaseString, just for the purpose of performing a contains_key check. Is there a better way to do this?